config/crd/bases/req.nephio.org_interfaces.yaml

```diff
@@ -34,6 +34,14 @@ spec:
           spec:
             description: TBD how do we distinguish the loopback from the vnic(s)
             properties:
+              addressing:
```

Comment:
K8s uses ipFamilyPolicy and ipFamilies for Service: https://kubernetes.io/docs/reference/kubernetes-api/service-resources/service-v1/#ServiceSpec

Can/should we follow the same practice?

Reply:
I aligned with the IPFamily convention. The IPFamily Policy is slightly different, but I reused the naming